src/sidebar/components/annotation-action-bar.js

```diff
@@ -77,10 +90,15 @@ function AnnotationActionBar({
 
   return (
     <div className="annotation-action-bar u-layout-row">
-      {showEditAction && <Button icon="edit" title="Edit" onClick={onEdit} />}
+      {showEditAction && (
+        /** @ts-ignore - Fix Button component type */
+        <Button icon="edit" title="Edit" onClick={onEdit} />
```

Comment:
Given how widely `Button` is used, can I suggest that you update the JSDoc on that component first and then rebase this?